src/components/Filters/FiltersList.vue

```diff
@@ -131,5 +136,10 @@ export default {
       this.$emit('onClearFilters')
     },
   },
+  computed: {
+    activeTab() {
+      return this.$route.path.split('search/')[1] || 'image'
+   },
```

Comment:
I will change how we detect the 'active tab' in the future, but for now this is good. thank you!